mltb2/openai.py

```diff
@@ -19,8 +19,9 @@
 
 import tiktoken
 import yaml
-from openai import AsyncAzureOpenAI, AsyncOpenAI, AzureOpenAI, OpenAI
+from openai import AsyncAzureOpenAI, AsyncOpenAI, AzureOpenAI, OpenAI 
```

Comment:
The trailing blankline should be removed. You can do this with `make format`. Please also see https://github.com/telekom/mltb2/blob/main/CONTRIBUTING.md#testing-linting-and-formatting